tests/test_user_login_unit.py

```diff
@@ -0,0 +1,32 @@
+"""
+Unit tests for user authentication utility
+"""
+
+def check_password_changed(user, new_pass) -> bool:
+    return user.check_password(new_pass)
+
+class TestChangePasswordUnit:
+    def test_change_password_success(self, unregistered_user):
+        """
+        Checks if password was changed correctly
+        """
+        new_pass = "new password"
+        unregistered_user.set_password(new_pass)
+        assert check_password_changed(unregistered_user, new_pass)
+
+    def test_change_wrong_password(self, unregistered_user):
+        """
+        Checks if a wrong password is accepted
+        """
+        new_pass = "new password"
+        unregistered_user.set_password(new_pass)
+        random_str = "random string"
+        assert not check_password_changed(unregistered_user, random_str)
+
+    def test_change_empty_password(self, unregistered_user):
+        """
+        Checks if an empty password was changed correctly
+        """
+        new_pass = ""
+        unregistered_user.set_password(new_pass)
+        assert check_password_changed(unregistered_user, new_pass)
```

Comment:
These methods have to pass in `self` but they don't actually use them. There's nothing wrong with that, but `pylint` will complain, so we have to make it shut up.

If you run either `pylint tests/test_user_login_unit.py` or alternatively `sh lint.sh`, you'll see that pylint is throwing this error:
    - `tests/test_user_login_unit.py:9:4: R0201: Method could be a function (no-self-use)`

The fix is easy, since the whole module needs this fix we can just add it at the top. The syntax for manually disabling a lint is this:
- `# pylint: disable=lint-name`

so for this file we can slap on a: `#pylint: disable=no-self-use` (and if you look at the header of almost all of the other tests, they also have this on there). 

The lint disables should come in before anything (even before the module docstring) and usually should have a comment under them telling the reader _why_ the lint is disabled.

For reference you can look at the other files in `tests/`. In any case, running the linter locally will show these errors before they get to the CI pipeline.